app/interactors/file_attachments/update_interactor.rb

```diff
@@ -76,14 +76,9 @@ def attachment_params
   def blob_attributes(file_attachment_revision)
```

Comment:
I think this param is provided by the context.

Reply:
It's not set at that point, which avoids setting it twice (once as the current revision, second as the updated one) - but I think we can avoid the need for this method call to hopefully make this moot.